app/models/office.rb

```diff
@@ -0,0 +1,7 @@
+class Office < ApplicationRecord
+  acts_as_mappable lat_column_name: :latitude, lng_column_name: :longitude
```

Comment:
I wish we could set these in the geokit-rails configuration initializer, but… 'tis not to be.